contents/docs/cdp/build/tutorial.md

```diff
@@ -44,13 +44,19 @@ And our config would look like:
 
 For information on what code to write and what special functions to use, check out [the overview](/docs/cdp/build) and [the developer reference](/docs/cdp/build/reference).
 
-### Using the transformation source editor
+### Using the source code editor
 
-Go to Data pipeline -> Manage apps tab -> Install app (advanced) -> Start coding.
+Add "pipeline-ui" feature flag and enable it. For destinations you'll need Data Pipelines add-on to be enabled.
 
-![App editor location](https://res.cloudinary.com/dmukukwp6/image/upload/v1712101259/posthog.com/contents/images/docs/cdp/install-app.png)
+Go to Data pipeline 3000 -> Apps Management tab -> enter a name and pick the app kind
```

Comment:
the path for this file makes it look like external-facing docs

we should release the thing if it's ready to go in external facing docs rather than tell people to enable flags

if this is about advice for internal staff, we could replace the "pipeline" section for internals when the flag is enabled and then the advice internally is we turn the flag off if we need to.

i always prefer when things make it more painful when we're not shipping something so we have an extra push to ship the thing 😊 🚢 😊 🚢 😊 🚢 😊 🚢

Reply:
That's the plan, it's just not 100% ready atm, need like 2 more PRs coming this week

Reply:
> It is unclear how a user would "Add "pipeline-ui" feature flag"

that's for local dev only - so you'd just add "pipeline-ui" there and enable it